versioned.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
+package csi;
+
+service Versioned {
+  rpc GetSupportedVersions (GetSupportedVersionsRequest)
+    returns (GetSupportedVersionsResponse) {}
+}
+message GetSupportedVersionsRequest {
+}
+
+message GetSupportedVersionsResponse {
+  // All the CSI versions that the Plugin supports. This field is
+  // REQUIRED.
+  repeated Version supported_versions = 1;
```

Comment:
Since we're changing this, we've found that we need to know what the min and max supported version so that this can be negotiated.